packages/component-library/src/CardList/CardList.js

```diff
@@ -0,0 +1,202 @@
+    listStyleType: "none",
+    alignSelf: "stretch"
+  }
+}));
+
+const checkTags = (entryTags, activeTags) => {
+  if (!entryTags) return false;
+  for (let i = 0; i < entryTags.length; i += 1) {
+    if (activeTags[entryTags[i]]) return true;
+  }
+  return false;
+};
+
+const CardList = ({ CardRegistry }) => {
+  const { entries, tags } = CardRegistry;
+
+  const initState = {};
+  Object.keys(tags).forEach(tag => {
+    initState[tag] = true;
+  });
+  const classes = useStyles();
+  const [mobileOpen, setMobileOpen] = useState(false);
+  const theme = useTheme();
+  const [activeTags, setActiveTags] = useState(initState);
+
+  function handleDrawerToggle() {
+    setMobileOpen(!mobileOpen);
+  }
+
+  const drawer = (
+    <div className={classes.toolbar}>
+      <h1>Filters</h1>
+      <h2>Topic</h2>
+      <ul>
+        {Object.keys(tags).map(tag => (
+          <li key={shortid.generate()} className={classes.filterItem}>
+            <Checkbox
+              value={activeTags[tag]}
+              label={tag}
+              onChange={() =>
+                setActiveTags({ ...activeTags, [tag]: !activeTags[tag] })
+              }
+            />
+          </li>
+        ))}
+      </ul>
+    </div>
+  );
+
+  return (
+    <div className={classes.root}>
+      <CssBaseline />
+      <AppBar position="fixed" className={classes.appBar}>
+        <Toolbar>
+          <IconButton
+            color="inherit"
+            aria-label="open drawer"
+            edge="start"
+            onClick={handleDrawerToggle}
+            className={classes.menuButton}
+          >
+            <MenuIcon />
+          </IconButton>
+          <Typography variant="h6" noWrap>
+            Responsive drawer
+          </Typography>
+        </Toolbar>
+      </AppBar>
+      <nav className={classes.drawer} aria-label="mailbox folders">
+        {/* The implementation can be swapped with js to avoid SEO duplication of links. */}
+        <Hidden smUp implementation="css">
+          <Drawer
+            variant="temporary"
+            anchor={theme.direction === "rtl" ? "right" : "left"}
+            open={mobileOpen}
+            onClose={handleDrawerToggle}
+            classes={{
+              paper: classes.drawerPaper
+            }}
+            ModalProps={{
+              keepMounted: true // Better open performance on mobile.
+            }}
+          >
+            {drawer}
+          </Drawer>
+        </Hidden>
+        <Hidden xsDown implementation="css">
+          <Drawer
+            classes={{
+              paper: classes.drawerPaper
+            }}
+            variant="permanent"
+            open
+          >
+            {drawer}
+          </Drawer>
+        </Hidden>
+      </nav>
+      <main className={classes.content}>
+        <h1>Did you know?</h1>
+        <div>
+          <ul className={classes.entriesList}>
+            {entries
+              .filter(entry => checkTags(entry.component.tags, activeTags))
```

Comment:
The behavior currently is that if you select 2 tags (say transportation and transit), then the cards are _expanded_ to include cards with _either_ tag. I'm not sure if that is the desired behavior or if _further narrowing down_ is the desired behavior. The latter feels more natural to me. @jaronheard, any thoughts?

Reply:
Hmm, I'm not totally sure. I think I'm going to throw this out in #team-design and see if anyone else has thoughts on this when it's one long list.

However, once you add categories, as shown in our Figma, then I think I have a clear idea of what is supposed to happen:
Select first tag in a category => Narrow selection
Select additional tags in a category => Expand selection

Reply:
That is the behavior here. So... it's as expected then? The dynamic still seems weird to me but I'm not strongly attached to it working one way or the other

Reply:
Examples:
_Selected Tags (category):_ What shows up
_Transportation (topic):_ Transportation
_Transportation (topic),  Housing (topic):_ Transportation OR Housing
_Transportation (topic), Bar Chart (visualization):_ Transportation AND Bar Chart
_Transportation (topic), Housing (topic), Bar Chart (visualization):_ (Transportation OR Housing)  AND Bar Chart
_Transportation (topic), Bar Chart (visualization), Portland (location):_ Transportation AND Bar Chart AND Portland